clients.json

```
@@ -159,6 +159,14 @@
     "authors": []
   },
 
+  {
+    "name": "godis",
+    "language": "Go",
+    "repository": "https://github.com/piaohao/godis",
+    "description": "redis client implement by golang, inspired by jedis.",
+    "authors": ["piaohao"]
```

This should be a Twitter handle. Removing.

```suggestion
    "description": "redis client implement by golang, inspired by jedis."
```